packages/create-react-app/createReactApp.js

```diff
@@ -40,6 +40,7 @@ const dns = require('dns');
 const envinfo = require('envinfo');
 const execSync = require('child_process').execSync;
 const fs = require('fs-extra');
+const isInstalledGlobally = require('is-installed-globally');
```

Comment:
Is there a simple way to achieve this without an extra dependency?

Reply:
We need to get a current global path for `npm` or `yarn` and check it against the current executing path for `npm` or `yarn`.

Not sure if it possible internally without dependency in create-react-app script.